src/main/java/mate/model/Driver.java

```diff
@@ -6,12 +6,36 @@
     private Long id;
     private String name;
     private String licenseNumber;
+    private String login;
+    private String password;
 
     public Driver(String name, String licenseNumber) {
```

Comment:
I think we should add toString() method here(to display valid data in exception messages)